prowler/providers/azure/services/defender/defender_service.py

```diff
@@ -136,6 +137,25 @@ def __get_security_contacts__(self):
                         )
                     }
                 )
+            except HttpResponseError as error:
+                if error.status_code == 404:
+                    security_contacts[subscription_name].update(
+                        {
```

Comment:
Please, add a test for this use case in the check `defender_additional_email_configured_with_a_security_contact`.